src/pages/en/editor-setup.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Editor Setup
+description: Set up your editor to build with Astro.
+---
+You can build your Astro site with any code editor, installed locally or in the cloud!
+
+[VS Code](https://code.visualstudio.com) is a popular code editor that can be dowloaded locally, and is also used by online platforms such as [GitHub.dev](https://github.dev), [GitHub Codespaces](https://github.com/features/codespaces) and [Gitpod](https://gitpod.io).
+
+
+## VS Code Extension
```

Comment:
Might it make sense to move the second paragraph below the VS Code header? Something like 

```suggestion
You can build your Astro site with any code editor, installed locally or in the cloud!

This page explains how to improve the experience of editing `.astro` files in supported editors.

## VS Code Extension

[VS Code](https://code.visualstudio.com) is a popular code editor that can be downloaded locally, and is also used by online platforms such as [GitHub.dev](https://github.dev), [GitHub Codespaces](https://github.com/features/codespaces) and [Gitpod](https://gitpod.io).
```

Reply:
Will do!